README.md

```diff
@@ -112,115 +112,74 @@ On **Ubuntu**, you need to install the following packages:
+As not all tests in the stock regression suite pass, run `oqs-test/run_tests.sh` instead of simply executing `make tests` to ensure the build was successful.
 
-So, in summary, if OpenSSL is installed in a default location and `oqs-openssh` is to be installed in `/opt/openssh` this command builds and installs `oqs-openssh`: `export OPENSSH_INSTALL=/opt/openssh && autoreconf && ./configure --with-libs=-lm --prefix=$OPENSSH_INSTALL --sysconfdir=$OPENSSH_INSTALL --with-liboqs-dir=`pwd`/oqs && make && make install`
-
-As not all stock `openssh` tests are passing, be sure to execute `oqs-test/run_tests.sh` instead of simply executing `make tests` to ensure the build was successful.
-
-To execute a connection test with one of the supported quantum-safe algorithms (chosen at random), run `python3 oqs-test/try_connection.py`. If all algorithms should be exercized, pass a parameter to this command, e.g., like this: `python3 oqs-test/try_connection.py all`. Be aware that this test can take a long time due to the number of algorithm combinations available.
+To execute a connection test with a randomly chosen key-exchange and signature algorithm, run `python3 oqs-test/try_connection.py`. If it is desired that all such combinations should be tested, run `python3 oqs-test/try_connection.py all`. Be aware that this test can take a long time due to the number of algorithm combinations available.
 
 ### Running OQS-OpenSSH
 
 The following instructions explain how to establish an SSH connection that uses quantum-safe key exchange and authentication.
 
 #### Generating quantum-safe authentication keys
 
-To setup quantum-safe authentication, the server (and optionally, the client) need to generate quantum-safe keys. In what follows, `<SIG>` is one of the quantum-safe digital signature algorithms listed in [Supported Algorithms](#supported-algorithms) section above.
-
-The server generates its key files with the right permissions, and then generates its key pair:
-
-	mkdir ~/ssh_server/
-	chmod 700 ~/ssh_server/
-	touch ~/ssh_server/authorized_keys
-	chmod 600 ~/ssh_server/authorized_keys
-	<path-to-openssh>/bin/ssh-keygen -t ssh-<SIG> -f ~/ssh_server/id_<SIG>
-
-To enable client-side public-key authentication, the client generates its key pair:
-
-	mkdir ~/ssh_client/
-	<path-to-openssh>/bin/ssh-keygen -t ssh-<SIG> -f ~/ssh_client/id_<SIG>
-
-The server then adds the client's public key to its authorized keys
-
-	cat ~/ssh_client/id_<SIG>.pub >> ~/ssh_server/authorized_keys
+To setup quantum-safe authentication, the server (and optionally, the client) need to generate quantum-safe keys. To generate keys for all the OQS algorithms supported by fork, simply run `make tests -e LTESTS=""`.
 
 #### Establishing a quantum-safe SSH connection
 
-In one terminal, run a server:
+In one terminal, run the ssh server:
 
-	<path-to-openssh>/sbin/sshd -p 2222 -d \
+	<path-to-openssh>/sbin/sshd -D \
+	                            -f <absolute-path-to>/regress/sshd_config \
```

Comment:
Is the "\<absolute-path-to\>" wording sufficiently clear? What about stating that this should be the `openssh` build directory? If so, what about replacing with `pwd`, i.e., "...run the ssh server from within the build directory : \<path-to-openssh-install-directory\>/sbin/sshd -D -f \`pwd\`/regress/sshd_config...."